scripts/footprint/size_report

```diff
@@ -318,11 +358,31 @@ def generate_target_memory_section(
 def print_tree(data, total, depth):
     base = os.environ['ZEPHYR_BASE']
     totp = 0
+
+    bcolors_linux = {
```

Comment:
bcolors_unix ?

Reply:
Done. (bcolors_ansi though as that is more precise (The escape codes are ANSI escape codes))